Documentation/network-policies.md

```diff
@@ -13,14 +13,23 @@ To keep your cluster safer, it's strongly recommended to enable network policies
 In this example we are closing all the inbound communication on the namespace monitoring and just allowing the necessary traffic.
 **This example are only tested with calico provider.**
```

Comment:
This example will close all inbound communication on the namespace monitoring, and allow only necessary traffic. 
**This example has been tested only with the Calico provider.**